example/src/screens/GraphPage.tsx

```diff
@@ -28,6 +33,8 @@ export function GraphPage() {
     hapticFeedback('impactLight')
   }, [])
 
+  const highestDate = useMemo(() => points[POINTS - 1]!.date, [])
```

Comment:
Is it possible that this could be null in some cases? Do we need to check that? A `!` is always a bit risky...